sass/_variables.scss

```diff
@@ -17,11 +17,15 @@ $white-transluant:       rgba($white, .8) !default;
 // Brand colors
 // -------------------------
 
-$brand-primary:         #f4d430 !default;
+// Could also be #3EA0DA, depending on HBP identity renewal
+$brand-primary:         #82BBC4 !default; // was F4D430 in v1
```

Comment:
I missed the list and didn't read the commit messages because I was thinking they will show up more visually in the pull request.